src/App.css

```diff
@@ -1,38 +1,43 @@
 .App {
   text-align: center;
+  margin-top: 1rem;
 }
 
-.App-logo {
-  height: 40vmin;
-  pointer-events: none;
+.user-info{
+  display: flex;
+  justify-content: space-evenly;
+  background-color: #03c6c6;
+  width:80%;
+  border-radius: 10px;
```

Comment:
Can you sort the keys alphabetically? It's not a compulsion but a good practice to be followed.